BETA.md

```diff
@@ -27,13 +27,10 @@ The VSCode extension is not yet published on the Marketplace, so you'll need to
 build it locally.
 
 * Make sure you have installed `node`, `npm` and VS Code.
-* Install the
-  ["Scala Syntax"](https://marketplace.visualstudio.com/items?itemName=daltonjorge.scala)
-  plugin in VS Code.
```

Comment:
Installing the metals VSCode extension now automatically ensures that https://marketplace.visualstudio.com/items?itemName=scala-lang.scala is installed.